FreeStickers.plugin.js

```diff
@@ -3781,7 +3783,7 @@ BdApi.Patcher.instead('FreeStickers', StickerSendabilityModule, isSendableSticke
         return true;
     }
 
-    if(stickerSendability !== StickerSendability.NONSENDABLE) {

```

Comment:
Since `StickerSendability.NONSENDABLE` was always undefined, this was always `true`. However, it should be a positive comparison.